README.md

```diff
@@ -6,7 +6,7 @@ Hi,
 
 This is a training repository for Kottans [frontend](https://github.com/Kottans/frontend),
 [backend](https://github.com/Kottans/backend),
-and [git](https://github.com/Kottans/git-course) courses. 
+and [git](https://github.com/Kottans/git-course) courses.
```

Comment:
There is no need to make changes in several lines of this file if you don't want to resolve a lot of conflicts